### ISSUE TITLE
vm2 is undefined error

### ISSUE THREAD

**Issue by williamkray** [vector-im/chatterbox#79]
**Describe the bug**
Attempt to run the application locally with `yarn start` resulted in an error in the chatterbox window when starting a new chat.

Full error:
```
Something went wrong…
vm2 is undefined

This occurred while running mountView@http://localhost:3000/node_modules/.vite/deps/hydrogen-view-sdk.js?v=37b91674:4779:12.

render/<@http://localhost:3000/src/ui/views/ChatterboxView.ts:13:65
mapView/<@http://localhost:3000/node_modules/.vite/deps/hydrogen-view-sdk.js?v=37b91674:5231:31
_addReplaceNodeBinding@http://localhost:3000/node_modules/.vite/deps/hydrogen-view-sdk.js?v=37b91674:5179:26
mapView@http://localhost:3000/node_modules/.vite/deps/hydrogen-view-sdk.js?v=37b91674:5220:17
render@http://localhost:3000/src/ui/views/ChatterboxView.ts:13:9
mount@http://localhost:3000/node_modules/.vite/deps/hydrogen-view-sdk.js?v=37b91674:5027:25
mountView@http://localhost:3000/node_modules/.vite/deps/hydrogen-view-sdk.js?v=37b91674:4777:17
view@http://localhost:3000/node_modules/.vite/deps/hydrogen-view-sdk.js?v=37b91674:5217:12
mapView/<@http://localhost:3000/node_modules/.vite/deps/hydrogen-view-sdk.js?v=37b91674:5233:21
binding@http://localhost:3000/node_modules/.vite/deps/hydrogen-view-sdk.js?v=37b91674:5184:35
update@http://localhost:3000/node_modules/.vite/deps/hydrogen-view-sdk.js?v=37b91674:5051:9
_updateFromValue@http://localhost:3000/node_modules/.vite/deps/hydrogen-view-sdk.js?v=37b91674:4976:10
emit/<@http://localhost:3000/node_modules/.vite/deps/hydrogen-view-sdk.js?v=37b91674:10481:32
emit@http://localhost:3000/node_modules/.vite/deps/hydrogen-view-sdk.js?v=37b91674:10481:16
emitChange@http://localhost:3000/node_modules/.vite/deps/hydrogen-view-sdk.js?v=37b91674:20356:12
_showTimeline@http://localhost:3000/src/viewmodels/RootViewModel.ts:62:10
_setupNavigation/<@http://localhost:3000/src/viewmodels/RootViewModel.ts:29:74
emit@http://localhost:3000/node_modules/.vite/deps/hydrogen-view-sdk.js?v=37b91674:1082:8
emitIfChanged@http://localhost:3000/node_modules/.vite/deps/hydrogen-view-sdk.js?v=37b91674:20043:12
applyPath@http://localhost:3000/node_modules/.vite/deps/hydrogen-view-sdk.js?v=37b91674:19928:48
push@http://localhost:3000/node_modules/.vite/deps/hydrogen-view-sdk.js?v=37b91674:19914:17
completeRegistration@http://localhost:3000/src/viewmodels/AccountSetupViewModel.ts:70:21
```

**To Reproduce**
1. clone repository
2. modify public/config.json with server information, token information, invite_user value
3. execute `yarn start` to run test server
4. visit localhost:3000 and begin chat
5. chat window returned error above
6. **invite_user user is successfully invited to a chat with the newly generated user**, but web-chat user cannot interact with the conversation

**Screenshots**
![image](https://user-images.githubusercontent.com/8409433/178841134-416d2e4f-9bd8-4b57-9481-b7f74eba7bf3.png)
![image](https://user-images.githubusercontent.com/8409433/178841524-f15439e2-f5f2-4dc1-900e-d964a1e7db39.png)

**Desktop (please complete the following information):**
 - OS: Arch Linux
 - Browser Firefox
 - Version 102.0.1

**Additional context**
All changes I've made are contained in the config.json. Full diff is below:
```diff
diff --git a/public/config.json b/public/config.json
index 62a90b9..53d1cba 100644
--- a/public/config.json
+++ b/public/config.json
@@ -1,18 +1,18 @@
 {
-    "homeserver": "https://your-homeserver.com",
+    "homeserver": "https://synapse.mssj.me",
     "auto_join_room": "Internal matrix id of the room",
     "username_prefix": "foobar",
     "header": {
         "title": "Your Fancy Title",
-        "avatar": "https://link_to_avatar/awesome.png"
+        "avatar": "https://www.mssj.me/images/logo.png"
     },
     "footer": {
         "chatterbox_link": "https://element.io/solutions/chatterbox-embedded-live-chat-for-customer-service",
         "matrix_link": "https://matrix.org"
     },
-    "token": "Valid Token to be used with Token Authenticated Registration",
-    "invite_user": "@botuser:your-homeserver.com",
+    "token": "REDACTED",
+    "invite_user": "REDACTED",
     "encrypt_room": false,
-    "avatar": "https://link_to_avatar/avatar.png",
+    "avatar": "https://www.mssj.me/images/logo.png",
     "disable_composer_until_operator_join": true
 }
 ```

**Comment by paulvt** [comment]
For me this was fixed by #82!

**Comment by williamkray** [comment]
ah yes, it seems to be resolved for me. closing!
